src/html/menu/content/bookmarks/content.html

```diff
@@ -102,7 +102,7 @@ <h1 class="menu-item-header-text">Content</h1>
         <span class="label-icon"></span>
         <span class="label-block">
           <span class="label-block-item">Align vertically</span>
-          <span class="label-block-item small muted">Works well with Bookmark List Style.</span>
+          <span class="label-block-item small muted">Works well with Bookmark Book Style.</span>
```

Comment:
I think it should be "...Block style". 

: )

Reply:
Yeah. Sorry!